tests/driver_sx127x/main.c

```diff
@@ -270,6 +272,39 @@ int listen_cmd(int argc, char **argv)
     return 0;
 }
 
+int syncword_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        puts("usage: syncword <get|set>");
+        return -1;
+    }
+
+    netdev_t *netdev = (netdev_t *)&sx127x;
+    uint8_t syncword;
+    if (strstr(argv[1], "get") != NULL) {
+        netdev->driver->get(netdev, NETOPT_SYNCWORD, &syncword,
+                            sizeof(syncword));
+        printf("Syncword: %02x\n", (int)syncword);
+        return 0;
+    }
+
+    if (strstr(argv[1], "set") != NULL) {
+        if (argc < 3) {
+            puts("usage: syncword set <syncword>");
+            return -1;
+        }
+        syncword = atoi(argv[2]);
```

Comment:
It's very counter intuitive regarding the `get` subcommand: here the input value is interpreted as an int but the `get` subcommand returns it as an hex.
So it's either everything is entered/printed as int or everything is entered/printed as hex.